app/views/events/_form.html.slim

```diff
@@ -8,44 +8,32 @@
       ul
         - event.errors.full_messages.each do |message|
           li= message
-
-  .form-inline
-    .form-group
-      = f.label :'Event Name: '
-      = f.text_area :event_name, class: "form-control", placeholder: "name of the event"
-    ' 
-    .form-group
-      = f.label :'Location: '
-      ' 
-      = f.text_area :location, class: "form-control", placeholder: "location of the event (including postal code)"
-    ' 
-    .form-group
-      = f.label :'Number of Volunteers: '
-      ' 
-      = f.number_field :number_of_volunteers, class: "form-control", placeholder: "total number of volunteers required"
-  br
-
   .row
     .form-group
-      .col-xs-10
-        = f.label :description
-        = f.text_area :description, class: "form-control", placeholder: "a short introduction of the event"
+      .col-xs-4
+        = f.label :'Event Name '
+        = f.text_field :event_name, class: "form-control", placeholder: "name of the event"
+      .col-xs-4
+        = f.label :'Location '
+        ' 
```

Comment:
remove this `'` and follow how [bootstrap](http://getbootstrap.com/css/#forms) is structuring the form